packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -1438,5 +1441,93 @@ describe('<Tabs />', () => {
       expect(hasLeftScrollButton(container)).to.equal(false);
       expect(hasRightScrollButton(container)).to.equal(false);
     });
+
+    // https://github.com/mui/material-ui/issues/41388
+    it('should allow a user to click a scrollable Tab without an error', async function test() {
+      function DynamicScrollableTabs() {
+        const [value, setValue] = React.useState(0);
+        const handleChange = (_, newValue) => {
+          setValue(newValue);
+        };
+        const tabLabels = [
+          'Item One',
+          'Item Two',
+          'Item Three',
+          'Item Four',
+          'Item Five',
+          'Item Six',
+          'Item Seven',
+          'Item Eight',
+          'Item Nine',
+          'Item Ten',
+        ];
+        return (
+          <Container maxWidth="sm">
+            <Box sx={{ width: '100%', m: 2 }}>
+              <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
+                <Tabs
+                  onChange={handleChange}
+                  value={value}
+                  variant="scrollable"
+                  scrollButtons
+                  allowScrollButtonsMobile
+                  selectionFollowsFocus
+                >
+                  {tabLabels.map((label, index) => (
+                    <Tab
+                      key={`tab-${index}`}
+                      label={label}
+                      id={`dynamic-tab-${index}`}
+                      aria-controls={`dynamic-tabpanel-${index}`}
+                    />
+                  ))}
+                </Tabs>
+              </Box>
+              {tabLabels.map((label, index) => {
+                return (
+                  <Box
+                    key={`tab-panel-${index}`}
+                    role="tabpanel"
+                    hidden={value !== index}
+                    id={`dynamic-tabpanel-${index}`}
+                    aria-labelledby={`dynamic-tab-${index}`}
+                  >
+                    {value === index && (
+                      <Box sx={{ p: 3 }}>
+                        <p>{label}</p>
+                      </Box>
+                    )}
+                  </Box>
+                );
+              })}
+            </Box>
+          </Container>
+        );
+      }
+
+      const { getByRole } = render(<DynamicScrollableTabs />);
+
+      const user = userEvent.setup();
+
+      // Tab and TabPanel for Item One should be visible
+      const tab1 = getByRole('tab', { name: 'Item One' });
+      expect(tab1).toBeVisible();
+      const tabPanel1 = getByRole('tabpanel', { name: 'Item One' });
+      expect(tabPanel1).toBeVisible();
+
+      // Tab for Item Seven should be visible
+      const tab7 = getByRole('tab', { name: 'Item Seven' });
+      expect(tab7).toBeVisible();
+
+      // Click on the Item Seven Tab
+      // In the test fixture, this fails with
+      //  TypeError: Cannot read properties of null (reading 'offsetHeight')
+      await user.click(tab7);
```

Comment:
This test is analogous to https://github.com/nicholeuf/mui-test-fixture-41388/blob/main/app/page.test.tsx. The user click action fails in the live example test fixture and succeeds here.